src/components/security-analysis-result.js

```diff
@@ -211,11 +216,31 @@ const SecurityAnalysisResult = ({ onClickNmKConstraint, result }) => {
         },
         [onClickNmKConstraint, classes.button]
     );
+
+    const toolTipValueGetterValues = (params) => {
+        if (
+            params?.data?.contingencyId &&
```

Comment:
Do you really need all the optional chaining?
If yes, you need to add "?" for : params.data?.contingencyEquipmentsIds